FirebaseAuth/Sources/Swift/AuthProvider/PhoneAuthProvider.swift

```diff
@@ -152,16 +159,13 @@ import Foundation
     open func verifyPhoneNumber(with multiFactorInfo: PhoneMultiFactorInfo,
                                 uiDelegate: AuthUIDelegate? = nil,
                                 multiFactorSession: MultiFactorSession?) async throws -> String {
-      return try await withCheckedThrowingContinuation { continuation in
-        self.verifyPhoneNumber(with: multiFactorInfo,
-                               uiDelegate: uiDelegate,
-                               multiFactorSession: multiFactorSession) { result, error in
-          if let error {
-            continuation.resume(throwing: error)
-          } else if let result {
-            continuation.resume(returning: result)
-          }
-        }
+      do {
+        multiFactorSession?.multiFactorInfo = multiFactorInfo
+        return try await verifyPhoneNumber(multiFactorInfo.phoneNumber,
+                                           uiDelegate: uiDelegate,
+                                           multiFactorSession: multiFactorSession)
+      } catch {
```

Comment:
No need to catch and rethrow. The function specifies `throws` so it this is effectively a no-op